### ISSUE TITLE
Xcode 10.2.1 运行时崩溃

### ISSUE THREAD

**Issue by z234009184** [HDB-Li/LLDebugTool#77]
崩溃在了一个分类中，好像布局部分有问题（方法找不到）。

**Comment by HDB-Li** [comment]
@z234009184  Hi, Is this error in `NSObject+LL_Hierarchy.h` at line 1236?

If yes, you can fix it by fork it, or waiting for next release version. I will release a new version this afternoon. 

If not this line, you can provide a more informations.

```
#ifdef __IPHONE_13_0
                if (@available(iOS 13.0, *)) {
                    weakSelf.activityIndicatorViewStyle = index + (UIActivityIndicatorViewStyleMedium - UIActivityIndicatorViewStyleGray - 1);
                }
#endif
```

**Comment by z234009184** [comment]
@HDB-Li Hi, Thank you for your reply

Yes, I have solved this problem manually, but this is a compilation problem. There is also a runtime crash problem. The error reporting "unrecognized selector sent to instance". Now I have reduced the version to 1.2.2

**Comment by HDB-Li** [comment]
@z234009184  Could you provider more infomations about `There is also a runtime crash problem` when you have a rest time. My XCode is 11.2 now, so if you can provider more infomations, I will try to find it's a low XCode version crash or a runtime bug in my new Debugtool version.

**Comment by HDB-Li** [comment]
@z234009184  Try version 1.3.5, I just release it. If you still have any questions, feel free to contact me.

**Comment by z234009184** [comment]
@HDB-Li  Errors are still reported in version 1.3.5. The error information is as follows:
file: UIView+LLWidgetBorder.m
line:  41   `[self LL_layoutSubviews];`
crash info: -[UINavigationBar _applyConstrainedContentSizeIfNecessary]: unrecognized selector sent to instance 0x105015740

![image](https://user-images.githubusercontent.com/12725906/68367426-ae45db00-0170-11ea-9374-a6f1a0663853.png)

**Comment by HDB-Li** [comment]
@z234009184  So bad. I will try to find the reason.  

: (

**Comment by HDB-Li** [comment]
@z234009184  Hi, I can't replay your crash, but I find some other fix this problem with a dispatch_once block. You can try this, maybe it's useful.

```
@implementation UIView (LLWidgetBorder)

+ (void)load {
    static dispatch_once_t onceToken;
    dispatch_once(&onceToken, ^{
        [self LL_swizzleInstanceMethodWithOriginSel:@selector(layoutSubviews) swizzledSel:@selector(LL_layoutSubviews)];
    });
}
```

**Comment by HDB-Li** [comment]
Fix in Version 1.3.6

**Comment by HDB-Li** [comment]
@z234009184  Fix in version 1.3.6

**Comment by z234009184** [comment]
WOW! It's worked. Thanks so much !!
The new version brings a lot of powerful features, it's too awesome!!
